doc/doxygen/Makefile

```diff
@@ -10,13 +10,23 @@ doc: html
 
 # by marking html as phony we force make to re-run Doxygen even if the directory exists.
 .PHONY: html
-html:
+html: src/css/riot.css
 	( cat riot.doxyfile ; echo "GENERATE_HTML = yes" ) | doxygen -
 
 .PHONY: man
 man:
 	( cat riot.doxyfile ; echo "GENERATE_MAN = yes" ) | doxygen -
 
+ifneq (,$(shell which lessc))
```

Comment:
I don't expect the CSS to change much (unless we decide for a whole new design, in which case the LESS file might become unneeded, too). If it is, the review needs to take care, that the LESS file is updated, too.